src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -945,6 +948,21 @@ private ConfigComparisonResult compareConfigs() {
     return comparisonResult;
   }
 
+  private void askUserForName() {
+    TextInputDialog dialog = new TextInputDialog();
+    dialog.setTitle("Enter your name");
+    dialog.setHeaderText("For auditing purposes, enter the name(s) of everyone currently "
+        + "operating this machine.");
+    dialog.setContentText("Name:");
+    Optional<String> result = dialog.showAndWait();
```

Comment:
The cancel button doesn't properly cancel out of this, it just runs it with no name provided. Same problem with the X button in the corner.

Reply:
Yeah, I was using the cancel functionality while developing -- I can just hit `esc` and tabulation continues. I can adjust to stop tabulation instead.